Api/Domain/Entities/Income.cs

```diff
@@ -4,9 +4,9 @@ public class Income : IEntityBase
 {
     public Guid Id { get; set; } = Guid.NewGuid();
     public Guid UserId { get; set; }
-    public string Currency { get; set; } = string.Empty;
+    public string Currency { get; set; }
     public decimal Amount { get; set; }
-    public string Source { get; set; } = string.Empty;
+    public string Source { get; set; }
     public DateTime Date { get; set; }
     public DateTime CreatedAt { get; set; }
```

Comment:
please add a default value for create at attribute, like "Now"

Reply:
Sure